addon/index.js

```diff
@@ -25,6 +24,9 @@ const mod = new PageMod({
   onAttach: setupApp
 });
 
+if (!store.experiments) store.experiments = [];
```

Comment:
maybe I'm missing something, or maybe you're still working out this part of the code, but I don't actually see `store` declared anywhere